src/components/AdjustableSidebarWidth.vue

```diff
@@ -280,7 +280,6 @@ export default {
      */
     toggleScrollLock(lock) {
       const scrollLockContainer = document.getElementById(this.scrollLockID);
-      if (!scrollLockContainer) return;
```

Comment:
This is being removed because we are always guaranteed to have rendered an element with this ID, correct?

Reply:
So in this case, the component is already gone from the DOM, by the time the `AdjustableSidebarWidth` is destroyed, so we were not calling the unlock scripts.